invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html

```diff
@@ -147,277 +147,8 @@
   </section>
 
 
-  <div class="ui container">
-    <div class="ui relaxed grid mt-5">
-      <div class="two column row top-padded">
+  {% include "invenio_app_rdm/records/record_content.html" %}
```

Comment:
is the idea to make `record_content.html` overridable?

Reply:
I'd say embeddable instead of overridable. If you take a look at https://github.com/inveniosoftware/invenio-app-rdm/issues/2953, having the record content be separate would facilitate a tabbed interface for the landing page. There is probably more discussion needed on the specific interface....but I think this template change makes the templates more flexible in general.